test/LoggerAbstractServiceFactoryTest.php

```diff
@@ -291,7 +291,7 @@ public function testWillInjectWriterPluginManagerIfAvailable()
     public function testWillInjectProcessorPluginManagerIfAvailable()
     {
         $processors = new ProcessorPluginManager(new ServiceManager());
-        $mockProcessor = $this->getMock('Zend\Log\Processor\ProcessorInterface');
+        $mockProcessor = $this->getMockBuilder('Zend\Log\Processor\ProcessorInterface')->getMock();
```

Comment:
And its true. https://phpunit.de/manual/4.8/en/test-doubles.html and https://phpunit.de/manual/current/en/test-doubles.html